src/nb_jsonapi_client/common.py

```diff
@@ -99,18 +100,20 @@ def mark_invalid(self):
 
 def error_from_response(response_content):
     try:
-        error_str = response_content['errors'][0]['title']
+        # error_str = response_content['errors'][0]['title']
+        # error_str = f"{response_content['code']}: {response_content['message']}"
+        error_str = pprint.pformat(response_content)
     except Exception:
         error_str = '?'
     return error_str
 
 
 def jsonify_attribute_name(name):
-    return name.replace('__', '.').replace('_', '-')
+    return name.replace('__', '.')
```

Comment:
Original library did some hinky stuff changing "_" to "-". Doesn't cut it for NB.